platforms/src/Github/Providers-config.ts

```diff
@@ -43,7 +41,7 @@ providers = [
     threshold: "60",
   }),
   new GithubContributionActivityProvider({
-    threshold: MAX_CONTRIBUTION_DAYS.toString(),
+    threshold: "120",
```

Comment:
I think moving this var to githubClient makes sense, but could we still source it here? Just so we don't forget to update it if we ever change the threshold here.

Reply:
Yes, make sense